Makefile

```diff
@@ -1,6 +1,9 @@
 .PHONY: build lint test clean run_image build_image download-test-vectors clean-vectors \
 	setup-hive test-pattern-default run-hive run-hive-debug clean-hive-logs
 
+default: ethereum-package checkout-ethereum-package localnet
```

Comment:
`checkout-ethereum-package` already has `ethereum-package` as a dependency we don't need to have it here

Reply:
Fixed!